sway-core/src/semantic_analysis/namespace/trait_map.rs

```diff
@@ -792,7 +823,11 @@ impl TraitMap {
     }
 }
 
-fn are_equal_minus_dynamic_types(type_engine: &TypeEngine, left: TypeId, right: TypeId) -> bool {
+pub fn are_equal_minus_dynamic_types(
```

Comment:
Please change this to `pub(crate)` if this function is not being use anywhere outside of this crate.